controllers/producer.go

```diff
@@ -339,7 +341,8 @@ func spliceSourceSvc(sourceIp string, ipToSvcCache map[string]string, cacheLock
 	return "", nil
 }
 
-func spliceDestinationSvc(entry *data_accesslog.HTTPAccessLogEntry, sourceSvc string) string {
+func spliceDestinationSvc(entry *data_accesslog.HTTPAccessLogEntry, sourceSvc string, svcToIpsCache map[string][]string,
```

Comment:
as previously mentioned, we should pass the lock everywhere

Reply:
should not

Reply:
change in future